docs/development.md

```diff
@@ -191,19 +184,19 @@ osh/hub/scripts/osh-xmlrpc-client.py \
 
 ## Code path when submitting user scan
 
- 1. Code execution starts in client, for a specific command, e.g. [diff-build](https://gitlab.cee.redhat.com/covscan/covscan/blob/master/osh/client/commands/cmd_diff_build.py).
-   * Files are uploaded to server via [`upload_file` XML-RPC call](https://gitlab.cee.redhat.com/covscan/covscan/blob/master/osh/client/commands/shortcuts.py#L104).
-   * The XML-RPC call itself [is defined](https://gitlab.cee.redhat.com/covscan/covscan/blob/master/osh/hub/settings.py#L160) [in kobo](https://github.com/release-engineering/kobo/blob/master/kobo/django/upload/xmlrpc.py#L19).
- 2. Server code path starts in XML-RPC API at specific method for particular scan type, e.g. for [mock builds](https://gitlab.cee.redhat.com/covscan/covscan/blob/master/osh/hub/osh_xmlrpc/scan.py#L57).
- 3. There is a hierarchical structure for configuring data for scan in `osh/hub/errata/scanner.py`, for client scans this is [ClientScanScheduler](https://gitlab.cee.redhat.com/covscan/covscan/blob/master/osh/hub/errata/scanner.py#L374).
+ 1. Code execution starts in client, for a specific command, e.g. [diff-build](https://github.com/openscanhub/openscanhub/blob/main/osh/client/commands/cmd_diff_build.py).
```

Comment:
Please amend your previous commit and remove the changes of adding internal GitLab links from it.

Reply:
Check the new commit please @siteshwar . I already fixed that

Reply:
I still see https://github.com/openscanhub/openscanhub/pull/29/commits/2e8d41a343276c0afb5f028c61c3fa3db21d2778 in your PR which adds links to internal GitLab.

Reply:
The last commit is [6857cf4](https://github.com/openscanhub/openscanhub/pull/29/commits/6857cf4136909c2719753818e78da4b18b62f3e9)

Reply:
You are going back and forth on changes. First you added references to internal links through https://github.com/openscanhub/openscanhub/pull/29/commits/2e8d41a343276c0afb5f028c61c3fa3db21d2778 and later switched them back to external links by https://github.com/openscanhub/openscanhub/pull/29/commits/6857cf4136909c2719753818e78da4b18b62f3e9. The changes in the first commit are unnecessary and should be amended.

Reply:
Sorry, but I am not understanding it. The last commit (https://github.com/openscanhub/openscanhub/pull/29/files) is the original file with just my new section on it.

Reply:
All referenced links there are from github

Reply:
@jperezdealgaba As I understand it, @siteshwar wanted you to squash the commits.  I have done it on your behalf.